packages/saml-auth/package.json

```diff
@@ -80,6 +80,7 @@
     "typescript": "5.4"
   },
   "dependencies": {
+    "react": "18.3.1",
```

Comment:
This is literally the fix.
It's a bit of a shame but we might have to match this manually to the version in each release of Moodlenet